### PR TITLE
update comments for documentation

### PR DESCRIPTION
**Pull request by jayongg**

### CODE REVIEW

File: src/MicrosoftTeams.ts [hunk 1]
```diff
@@ -49,23 +49,77 @@ namespace microsoftTeams {
...
+    /**
+     * Represents information about a tab instance
+     */
     export interface TabInstance {
+        /**
+         * The name of the tab
+         */
         tabName: string;
+
+        // internal: do not use
```
Comment (ydogandjiev) [inline src/MicrosoftTeams.ts]: Any particular reason why you aren't using standard jsdoc format for this comment as well?

File: src/MicrosoftTeams.ts [hunk 2]
```diff
@@ -49,23 +49,77 @@ namespace microsoftTeams {
...
         internalTabInstanceId?: string;
+
+        /**
+         * last viewed time of this tab. null means unknown
```
Comment (ydogandjiev) [inline src/MicrosoftTeams.ts]: NIT: Capitalize "last" for consistency with other comments

File: src/MicrosoftTeams.ts [hunk 3]
```diff
@@ -49,23 +49,77 @@ namespace microsoftTeams {
...
+         * The Microsoft Teams ID for the channel with which the content is associated.
+         */
         channelId?: string;
+
+        /**
+         * The name for the channel with which the content is associated.
+         */
         channelName?: string;
+
+        // is this tab in a favorite channel?
```
Comment (ydogandjiev) [inline src/MicrosoftTeams.ts]: Consider using jsdoc format throughout

File: src/MicrosoftTeams.ts [hunk 4]
```diff
@@ -49,23 +49,77 @@ namespace microsoftTeams {
...
+         */
         teamId?: string;
+
+        /**
+         * The name for the team with which the content is associated.
+         */
         teamName?: string;
+
+        /**
+         * is this tab in a favorite team?
```
Comment (ydogandjiev) [inline src/MicrosoftTeams.ts]: NIT: Capitalize "is" for consistency with other comments

File: src/MicrosoftTeams.ts [hunk 5]
```diff
@@ -49,23 +49,77 @@ namespace microsoftTeams {
...
         teamIsFavorite?: boolean;
+
+        /**
+         * The Office 365 group ID for the team with which the content is associated.
+         * This field is available only when the identity permission is requested in the manifest.
+         */
         groupId?: string;
+
+        /**
+         * URL of this tab
```
Comment (ydogandjiev) [inline src/MicrosoftTeams.ts]: "Content URL of this tab" might be a little more informative

File: src/MicrosoftTeams.ts (hunk 1 above)
Comment (jayongg) [reply src/MicrosoftTeams.ts]: because it was "private" but I can update.

**Pull request merged**